src/Callback.php

```diff
@@ -35,13 +35,12 @@ class Callback
     }
     use StaticAddToTrait;
 
-    /**
-     * Specify a custom GET trigger here.
-     *
-     * @var string|null
-     */
+    /** @var string Specify a custom GET trigger. */
     protected $urlTrigger;
 
+    /** @var bool Create app sticky trigger. */
+    public $isSticky = false;
```

Comment:
I think we should not use this flag and make everything sticky by default

Reply:
I have try with default to all sticky but then other problems come up. For instance, when running crud test, delete is not trigger after edit because of that.

Reply:
Hmm, I think this will not allow proper nesting, but all cases will work, fine with me for this PR